src/ObjectUtils/CoffFileTest.cpp

```diff
@@ -78,3 +79,18 @@ TEST(CoffFile, FileDoesNotExist) {
   EXPECT_THAT(absl::AsciiStrToLower(coff_file_or_error.error().message()),
               testing::HasSubstr("no such file or directory"));
 }
+
+TEST(CoffFile, LoadsPdbPathSuccessfully) {
+  // Note that our test library libtest.dll does not have a PDB file path.
+  std::filesystem::path file_path = orbit_test::GetTestdataDir() / "dllmain.dll";
+
+  auto coff_file_or_error = CreateCoffFile(file_path);
+  ASSERT_THAT(coff_file_or_error, HasNoError());
+
+  std::optional<PdbDebugInfo> pdb_debug_info = coff_file_or_error.value()->GetDebugPdbInfo();
+  ASSERT_TRUE(pdb_debug_info.has_value());
```

Comment:
would it be possible to add a test for missing debug info?

Reply:
Done.